distributed/comm/core.py

```diff
@@ -238,7 +238,7 @@ def resolve_address(addr):
 
 
 @gen.coroutine
-def connect(addr, timeout=3, deserialize=True):
+def connect(addr, timeout=3, deserialize=True, ssl_options=None):
```

Comment:
That's a good point. My initial idea was to specify the TLS configuration in the config file, which would have avoided passing extra arguments. Is it more convenient to specify the TLS configuration on the command line instead?